### PR TITLE
two new time formats

### LINKED ISSUE #282: Feature: two new time formats
add support "date_time":"1669116300.202074" and "2023/01/23 14:11:37" (nginx error logs)

### PR DESCRIPTION
**Pull request by DSmolonogov**
Fixes #282

### CODE REVIEW

File: plugin/action/convert_date/convert_date_test.go [hunk 1]
```diff
@@ -44,6 +44,42 @@ func TestConvert(t *testing.T) {
...
+	})
+
+	outEvents := make([]*pipeline.Event, 0)
+	output.SetOutFn(func(e *pipeline.Event) {
+		outEvents = append(outEvents, e)
+		wg.Done()
+	})
+
+	input.In(0, "test.log", 0, []byte(`{"time":998578999.1346}`))
+	input.In(0, "test.log", 0, []byte(`{"time":"2022/02/07 13:06:14"}`))
```
Comment (HeadHunter483) [inline plugin/action/convert_date/convert_date_test.go:71]: Maybe it would be better to add this tests to `TestConvert` func

**Review by HeadHunter483 — APPROVED** [review]
LGTM!

File: decoder/nginx.go [hunk 2]
```diff
@@ -9,7 +9,7 @@ import (
 )
 
 const (
-	nginxDateFmt = "2006/01/02 15:04:05"
+	NginxDateFmt = "2006/01/02 15:04:05"
 	parseDate    = false
```
Comment (vano144) [inline decoder/nginx.go:13]: do we need it?

**Review by vano144 — APPROVED** [review]

File: pipeline/util.go [hunk 3]
```diff
@@ -54,7 +56,10 @@ func StringToByteUnsafe(s string) (b []byte) {
 }
 */
 
-const formats = "ansic|unixdate|rubydate|rfc822|rfc822z|rfc850|rfc1123|rfc1123z|rfc3339|rfc3339nano|kitchen|stamp|stampmilli|stampmicro|stampnano"
+const (
+	formats   = "ansic|unixdate|rubydate|rfc822|rfc822z|rfc850|rfc1123|rfc1123z|rfc3339|rfc3339nano|kitchen|stamp|stampmilli|stampmicro|stampnano|timestamp|nginx_errorlog"
+	Timestamp = "timestamp"
```
Comment (vadimalekseev) [inline pipeline/util.go:61]: nit: Looks like we do not need to export `Timestamp` variable

**Pull request merged**